v4/source/jobs.yaml

```diff
@@ -0,0 +1,171 @@
+  '/jobs':
+    get:
+      tags:
+        - jobs
+      summary: Get the jobs.
+      description: |
+        Get a page of jobs. Use the query parameters to modify the behaviour of this endpoint.
+        __Minimum server version: 4.1__
+        ##### Permissions
+        Must have `manage_system` permission.
+      parameters:
+        - name: page
+          in: query
+          description: The page to select.
+          default: "0"
+          type: string
+        - name: per_page
+          in: query
+          description: The number of jobs per page.
+          default: "60"
+          type: string
+      responses:
+        '200':
+          description: Job list retrieval successful
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Job'
+        '400':
+          $ref: '#/responses/BadRequest'
+        '401':
+          $ref: '#/responses/Unauthorized'
+        '403':
+          $ref: '#/responses/Forbidden'
+
+    post:
+      tags:
+      - jobs
+      summary: Create a new job.
+      description: |
+        Create a new job.
```

Comment:
I realize it probably doesn't exist yet but later we should link to documentation about how jobs work